src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -236,7 +237,9 @@ init_state(Db, Fd, State, Header) ->
         view_states=ViewStates
     } = Header,
 
-    IdBtOpts = [{compression, couch_db:compression(Db)}],
+    IdBtOpts = [
+        {compression, couch_compress:get_compression_method()}
```

Comment:
This would work since old implementation calls to `couch_compress:get_compression_methods()` from `couch_db_updater:init_db`. However this would mean that we wouldn't be able to support different compression methods for different databases. Also it wouldn't be possible to delegate the decision on a compression algorithm to the storage engine (if we are not dispatching through `couch_engine`).